### ISSUE TITLE
Not all buttons are Buttons and they don't indicate disability apart from color in Dashboard screen.

### ISSUE THREAD

**Issue by rgarg24** [percussion/percussioncms#809]
**Describe the bug**
Buttons don't indicate disability outside color in Dashboard. They are displayed on screen but are not functional.

**To Reproduce**
1. Go to Dashboard screen "http://localhost:9992/cm/app/?view=dash&pathType=unknown#"
2. Click on different buttons on upper right such as "Launch Preview" etc.
3. It is not a button and it don't indicate disability outside color.

**Expected behavior**
Disability should be indicated and it should be buttons.
